src/smart-components/ChannelList/context/ChannelListProvider.tsx

```diff
@@ -150,10 +152,13 @@ const ChannelListProvider: React.FC<ChannelListProviderProps> = (props: ChannelL
     onBeforeCreateChannel,
     sortChannelList,
     overrideInviteUser,
-    disableAutoSelect,
+    activeChannelUrl,
```

Comment:
Isn't it breaking change or our public interface?
We should take one of the ways
1. Apply this change in the next minor version up & check Docs & notice to SE team & remove disableAutoSelect props from type definition
2. Keep both props

Reply:
no, its not... I import it two lines below.. is an augmentation ~

Reply:
`const disableAutoSelect = props?.disableAutoSelect || !!activeChannelUrl;`
Is a trick to use have same behavior if customer sets  an activeChannelUrl ~